packages/mui-material/src/Button/Button.js

```diff
@@ -78,14 +78,14 @@ const ButtonRoot = styled(ButtonBase, {
   ({ theme, ownerState }) => ({
     ...theme.typography.button,
     minWidth: 64,
+    padding: '6px 16px',
```

Comment:
is this line really needed if the default size will always be _md_?

Reply:
We don't have any other padding styles for md, and as the md is the default size, this is why the styles are defined here. Maybe I misunderstood your question :)